build.gradle

```diff
@@ -79,6 +79,12 @@ shadowJar {
     }
 }
 
+eclipse.classpath.file.whenMerged {classpath ->
+    classpath.entries.removeAll(classpath.entries.findAll {
+        entry -> entry.kind == 'lib' && entry.path.endsWith('com.typesafe/config/1.2.1/f771f71fdae3df231bcd54d5ca2d57f0bf93f467/config-1.2.1.jar')
```

Comment:
Surefire way to know that regardless of the hash that it's going to be excluded.